tap_hubspot/hubspot.py

```diff
@@ -223,6 +225,174 @@ def streams(self, start_date: datetime, end_date: datetime):
         else:
             raise NotImplementedError(f"unknown stream_id: {self.tap_stream_id}")
 
+    def get_contacts(
+        self, start_date: datetime, end_date: datetime
+    ) -> Iterable[Tuple[Dict, datetime]]:
+        filter_key = "lastmodifieddate"
+        obj_type = "contacts"
+        gen = self.search(
+            obj_type,
+            filter_key,
+            start_date,
+            end_date,
+            MANDATORY_PROPERTIES[f"{obj_type}"],
+        )
+
+        while True:
+            deals = {}
+            i = 0
+
+            for rec, ts in gen:
```

Comment:
this will break on the latest version of search that is not returning a tuple, but rather simply the record entry.